feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/data/network/coingecko/PriceApi.kt

```diff
@@ -4,23 +4,30 @@ import retrofit2.http.GET
 import retrofit2.http.Path
 import retrofit2.http.Query
 
-interface CoingeckoApi {
+interface PriceApi {
 
     companion object {
-        fun getRecentRateFieldName(coingeckoId: String): String {
-            return coingeckoId + "_24h_change"
+        fun getRecentRateFieldName(priceId: String): String {
+            return priceId + "_24h_change"
         }
     }
 
-    @GET("//api.coingecko.com/api/v3/coins/{id}/market_chart/range")
+    @GET("//tokens-price.novasama-tech.org/api/v3/coins/{id}/market_chart/range")
     suspend fun getCoinRange(
         @Path("id") id: String,
         @Query("vs_currency") currency: String,
         @Query("from") fromTimestamp: Long,
         @Query("to") toTimestamp: Long
     ): CoinRangeResponse
 
-    @GET("//api.coingecko.com/api/v3/simple/price")
+    @GET("//tokens-price.novasama-tech.org/api/v3/coins/{id}/market_chart")
+    suspend fun getLastCoinRange(
+        @Path("id") id: String,
+        @Query("vs_currency") currency: String,
+        @Query("days") days: String
+    ): CoinRangeResponse
+
+    @GET("//tokens-price.novasama-tech.org/api/v3/simple/price")
```

Comment:
I wonder whether we can just set the `baseUrl` during api creation to avoid dublicating it here